src/test/java/com/kainos/ea/service/RolesServiceTest.java

```diff
@@ -5,22 +5,33 @@
 
 import com.kainos.ea.database.DatabaseConnection;
 import com.kainos.ea.model.JobRole;
+import com.kainos.ea.trueApplication;
+import com.kainos.ea.trueConfiguration;
+import io.dropwizard.configuration.ResourceConfigurationSourceProvider;
+import io.dropwizard.testing.junit5.DropwizardAppExtension;
 import org.junit.jupiter.api.Test;
 import org.junit.jupiter.api.extension.ExtendWith;
 import org.mockito.Mockito;
 import org.mockito.junit.jupiter.MockitoExtension;
 
+import javax.ws.rs.client.Entity;
+import javax.ws.rs.core.MediaType;
+import javax.ws.rs.core.Response;
 import java.io.IOException;
 import java.sql.Connection;
 import java.sql.SQLException;
 import java.util.ArrayList;
 import java.util.List;
 
-import static org.junit.jupiter.api.Assertions.assertEquals;
-import static org.junit.jupiter.api.Assertions.assertThrows;
+import static org.junit.jupiter.api.Assertions.*;
 
 class RolesServiceTest {
 
+    static final DropwizardAppExtension<trueConfiguration> APP = new DropwizardAppExtension<>(
+            trueApplication.class, null,
+            new ResourceConfigurationSourceProvider()
+    );
```

Comment:
this shouldn't be in a unit test class, this is for integration tests

Reply:
removed